.circleci/verbatim-sources/linux-binary-build-defaults.yml

```diff
@@ -27,13 +29,11 @@ binary_linux_build: &binary_linux_build
   - run:
       name: Build
       no_output_timeout: "1h"
-      command: |
-        source "/pytorch/.circleci/scripts/binary_linux_build.sh"
+      command: ~/workspace/.circleci/scripts/binary_linux_build.sh
```

Comment:
@glenjamin my comment should really be placed here. If we keep this change then this other checkout of Pytorch https://github.com/pytorch/pytorch/blob/master/.circleci/scripts/binary_checkout.sh#L20-L38 should be removed, since the this job will call into the scripts downloaded in the circleci-checkout instead of these.

Now that I look at it more, would this work for the nightly case https://github.com/pytorch/pytorch/blob/master/.circleci/scripts/binary_checkout.sh#L34-L36 when we want to build the last Pytorch branch of the previous day rather than the current master?

Reply:
Aha, I'm with you now. You're right - this script is now building from the wrong checkout.

As this job has already run `binary_checkout` by this point, we don't need to use the script from the workspace.

i'll go back over the PR and check the script paths more closely.